src/App.js

```diff
@@ -17,17 +17,74 @@ class App extends React.Component {
       './img/9.png',
       './img/10.png',
     ],
+    defaultSelectOption: 3,
+    defaultStep: 3,
+    defaultInfinity: true,
   };
 
+  changeFrameSize = (e) => {
+    const newFrameSize = +e.target.value;
+
+    this.setState(() => ({ defaultSelectOption: newFrameSize }));
+  }
+
+  changeStep = (e) => {
+    const newStep = +e.target.value;
+
+    this.setState(() => ({ defaultStep: newStep }));
+  }
+
+  changeInfinity = () => {
+    this.setState(() => ({ defaultInfinity: !this.state.defaultInfinity }));
+  }
+
   render() {
-    const { images } = this.state;
+    /* eslint-disable-next-line max-len */
+    const { images, defaultSelectOption, defaultStep, defaultInfinity } = this.state;
```

Comment:
не відключай лінтер. В тудентів немає випадків коли його треба відключати, фікситься це ось так
```suggestion
    const {
      images, defaultSelectOption,
      defaultStep, defaultInfinity,
     } = this.state;
```